options/locale/locale_en-US.ini

```diff
@@ -1266,7 +1266,9 @@ pulls.merged_as = The pull request has been merged as <a rel="nofollow" class="u
 pulls.is_closed = The pull request has been closed.
 pulls.has_merged = The pull request has been merged.
 pulls.title_wip_desc = `<a href="#">Start the title with <strong>%s</strong></a> to prevent the pull request from being merged accidentally.`
-pulls.cannot_merge_work_in_progress = This pull request is marked as a work in progress. Remove the <strong>%s</strong> prefix from the title when it's ready
+pulls.cannot_merge_work_in_progress = This pull request is marked as a work in progress. 
+pulls.remove_wip_prefix = `<a href="#">Remove the <strong>%s</strong> prefix</a> from the title when it's ready` 
+pulls.add_wip_prefix = `Still in progress? <a href="#">Add <strong>%s</strong> prefix</a>`
```

Comment:
Why did you put these strings under backticks (``)?

Reply:
Because `pulls.title_wip_desc` also has these backticks so I assumed there must probably be a reason for that. Maybe the `#` is otherwise parsed as the start of a comment?

Reply:
I've removed the backticks with the removal of `#`.